### PR TITLE
Dịch "batch-norm_vn" - Phần 5

### LINKED ISSUE #1377: Dịch "batch-norm_vn" - Phần 5
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_convolutional-modern/batch-norm_vn.md#L216-L303

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by DacDinh147**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #1377

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 1]
```diff
@@ -219,23 +219,27 @@ apply the same $\hat{\mathbf{\mu}}$ and $\hat{\mathbf{\sigma}}$ to normalize the
...
 
 <!--
 As we mentioned earlier, BN typically behaves differently in training mode and prediction mode.
 First, the noise in $\mathbf{\mu}$ and $\mathbf{\sigma}$ arising from estimating each on minibatches are no longer desirable once we have trained the model.
 Second, we might not have the luxury of computing per-batch normalization statistics, e.g., we might need to apply our model to make one prediction at a time.
 -->
 
-*dịch đoạn phía trên*
+Như chúng tôi đã đề cập trước đó, BN thường hoạt động khác nhau trong chế độ huấn luyện và chế độ dự đoán.
+Thứ nhất, nhiễu trong $\mu$ và $\sigma$ phát sinh từ việc ước tính chúng trên những minibatch thì không còn là nhiễu mà chúng ta mong muốn khi chúng ta huấn luyện mô hình.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:231]:
```suggestion
Thứ nhất, nhiễu trong $\mu$ và $\sigma$ phát sinh từ việc chúng được xấp xỉ trên những minibatch không còn là nhiễu được mong muốn, một khi ta đã huấn luyện xong mô hình.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 2]
```diff
@@ -219,23 +219,27 @@ apply the same $\hat{\mathbf{\mu}}$ and $\hat{\mathbf{\sigma}}$ to normalize the
...
+Thứ hai, chúng ta không có tài nguyên xa xỉ để tính toán các con số thống kê trên mỗi lần chuẩn hoá theo batch, ví dụ: chúng ta cần áp dụng mô hình để đưa ra kết quả dự đoán tại một thời điểm.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:232]:
```suggestion
Thứ hai, chúng ta không có tài nguyên xa xỉ để tính toán các con số thống kê trên mỗi lần chuẩn hoá theo batch, ví dụ: chúng ta cần áp dụng mô hình để đưa ra một kết quả dự đoán mỗi lần.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 3]
```diff
@@ -298,7 +302,12 @@ The `num_features` parameter required by the `BatchNorm` instance is the number
 The `num_dims` parameter also required by this instance is 2 for a fully-connected layer and 4 for a convolutional layer.
 -->
 
-*dịch đoạn phía trên*
+Bây giờ chúng ta có thể tạo ra một tầng `BatchNorm` thích hợp.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:305]:
```suggestion
Bây giờ chúng ta có thể tạo ra một tầng `BatchNorm` đúng cách.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 4]
```diff
@@ -298,7 +302,12 @@ The `num_features` parameter required by the `BatchNorm` instance is the number
...
+Tầng này sẽ duy trì những tham số thích hợp tương ứng với tỉ lệ `gamma` và độ xê dịch `beta`, hai tham số này sẽ được cập nhật trong quá trình huấn luyện.
+Thêm vào đó, tầng BN sẽ sẽ duy trì một giá trị trung bình động của những giá trị trung bình và phương sai cho lần dùng tiếp theo trong việc dự đoán của mô hình.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:307]:
```suggestion
Thêm vào đó, tầng BN sẽ sẽ duy trì một giá trị trung bình động của những giá trị trung bình và phương sai cho các lần chạy sau trong lúc mô hình dự đoán. 
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 5]
```diff
@@ -298,7 +302,12 @@ The `num_features` parameter required by the `BatchNorm` instance is the number
 The `num_dims` parameter also required by this instance is 2 for a fully-connected layer and 4 for a convolutional layer.
 -->
 
-*dịch đoạn phía trên*
+Bây giờ chúng ta có thể tạo ra một tầng `BatchNorm` đúng cách.
+Tầng này sẽ duy trì những tham số thích hợp tương ứng với tỉ lệ `gamma` và độ xê dịch `beta`, hai tham số này sẽ được cập nhật trong quá trình huấn luyện.
```
Comment (DacDinh147) [inline chapter_convolutional-modern/batch-norm_vn.md:306]: sửa `độ xê dịch` thành `độ chệch` để thống nhất với các chương còn lại
```suggestion
Tầng này sẽ duy trì những tham số thích hợp tương ứng với tỉ lệ `gamma` và độ chệch `beta`, hai tham số này sẽ được cập nhật trong quá trình huấn luyện.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 6]
```diff
@@ -298,7 +302,12 @@ The `num_features` parameter required by the `BatchNorm` instance is the number
 The `num_dims` parameter also required by this instance is 2 for a fully-connected layer and 4 for a convolutional layer.
 -->
 
-*dịch đoạn phía trên*
+Bây giờ chúng ta có thể tạo ra một tầng `BatchNorm` đúng cách.
+Tầng này sẽ duy trì những tham số thích hợp tương ứng với tỉ lệ `gamma` và độ chệch `beta`, hai tham số này sẽ được cập nhật trong quá trình huấn luyện.
+Thêm vào đó, tầng BN sẽ sẽ duy trì một giá trị trung bình động của những giá trị trung bình và phương sai cho các lần chạy sau trong lúc mô hình dự đoán. 
+Tham số `num_features` được yêu cầu truyền vào bởi đối tượng `BatchNorm` là số lượng các đầu ra cho tầng kết nối đầy đủ và số lượng kênh đầu ra cho tầng tích chập.
+Tham số `num_dims` cũng được yêu cầu truyền vào bởi đối tượng này là 2 cho tầng kết nối toàn bộ và 4 cho tầng tích chập.
```
Comment (cuongvng) [inline chapter_convolutional-modern/batch-norm_vn.md:309]:
```suggestion
Tham số `num_dims` cũng được yêu cầu truyền vào bởi đối tượng này là 2 cho tầng kết nối đầy đủ và 4 cho tầng tích chập.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 7]
```diff
@@ -245,13 +249,13 @@ Recall that dropout also exhibits this characteristic.
 ## Implementation from Scratch
 -->
 
-## *dịch tiêu đề phía trên*
+## Lập trình từ đầu
```
Comment (duythanhvn) [inline chapter_convolutional-modern/batch-norm_vn.md:252]:
```suggestion
## Lập trình Từ đầu
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 8]
```diff
@@ -219,23 +219,27 @@ apply the same $\hat{\mathbf{\mu}}$ and $\hat{\mathbf{\sigma}}$ to normalize the
 ### Batch Normalization During Prediction
 -->
 
-### *dịch tiêu đề phía trên*
+### Chuẩn hoá theo batch trong quá trình dự đoán
```
Comment (duythanhvn) [inline chapter_convolutional-modern/batch-norm_vn.md:222]:
```suggestion
### Chuẩn hoá theo Batch trong Quá trình Dự đoán
```
